lib/anonymizer.js

```diff
@@ -15,7 +16,8 @@ const passwordRegex = /(?<=name="password"[\s\S]*?\r\n\r\n)([A-Za-z0-9]+[A-Za-z0
 
 //default
 let anonymizationEnabled = false;
-let manualMode = false;
+let manualMode = config.manualMode;
+let customAnonymizationRules = config.customAnonymizationRules || [];
```

Comment:
@AJun01 where is this `customAnonymizationRules` variable used?

Reply:
<img width="481" alt="image" src="https://github.com/user-attachments/assets/84413f64-9e6b-4d72-8a6f-ab91522418cc">
<img width="446" alt="image" src="https://github.com/user-attachments/assets/04ae8331-5ee0-444e-b8a2-a086dae14f23">